### ISSUE TITLE
CircleCI API Authentication Fails

### ISSUE THREAD

**Issue by matbrady** [pantheon-systems/terminus-build-tools-plugin#461]
## Problem

Unable to authenticate with CircleCI API using a Personal API Token when creating a new project. Example command:
```
terminus build:project:create --admin-password="***********" shortcut-name project-name
```

After providing a newly created API token, I'm continuously prompted with 
```
Call CircleCI API: https://circleci.com/api/v1.1/me
Your provided authentication token could not be used to authenticate with the CircleCI service. Please re-enter your credential.
```

<img width="529" alt="Screen_Shot_2022-09-21_at_12_43_06_PM" src="https://user-images.githubusercontent.com/1770862/191564073-156ea47d-3df3-44be-ba07-cb26681b64c1.png">

<img width="400" alt="Screen Shot 2022-09-21 at 12 49 29 PM" src="https://user-images.githubusercontent.com/1770862/191564452-87b95faa-2b94-45f6-be76-9c449a71c15f.png">

Circle CI API `/me` response ( https://circleci.com/api/v1.1/me)

<details>
<summary>API Response</summary>

```
{
  "enrolled_betas" : [ ],
  "in_beta_program" : false,
  "selected_email" : "REDACTED",
  "avatar_url" : "https://avatars.githubusercontent.com/u/REDACTED?v=4",
  "trial_end" : "2018-01-23T20:44:14.807Z",
  "admin" : null,
  "basic_email_prefs" : "smart",
  "sign_in_count" : 37,
  "github_oauth_scopes" : [ "user:email", "repo" ],
  "email_authentication" : null,
  "analytics_id" : "REDACTED",
  "name" : "REDACTED",
  "gravatar_id" : null,
  "first_vcs_authorized_client_id" : "REDACTED",
  "days_left_in_trial" : -1701,
  "privacy_optout" : false,
  "parallelism" : 1,
  "student" : false,
  "bitbucket_authorized" : false,
  "github_id" : REDACTED,
  "web_ui_pipelines_optout" : "opted-in",
  "bitbucket" : null,
  "dev_admin" : false,
  "all_emails" : [ "REDACTED", "REDACTED", "REDACTED" ],
  "created_at" : "2018-01-09T20:44:14.807Z",
  "plan" : null,
  "identities" : {
    "github" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/REDACTED?v=4",
      "external_id" : REDACTED,
      "id" : REDACTED,
      "name" : "REDACTED",
      "user?" : true,
      "domain" : "github.com",
      "type" : "github",
      "authorized?" : true,
      "provider_id" : "REDACTED",
      "login" : "REDACTED"
    }
  },
  "projects" : "REDACTED",
  "login" : "REDACTED",
  "organization_prefs" : { },
  "containers" : 1,
  "pusher_id" : "REDACTED",
  "web_ui_pipelines_first_opt_in" : true,
  "num_projects_followed" : 21
}
```

</details>

## Versions
- terminus version `3.0.8`
- terminus-build-tools-plugin version  `3.0.3`

**Comment by tedw** [comment]
Same issue here. Everything looks fine when I view https://circleci.com/api/v1.1/me as well as https://circleci.com/api/v2/me, though the v2 output looks like this:

```json
{
  "name" : "Ted Whitehead",
  "login" : "REDACTED",
  "id" : "REDACTED"
}
```

**Comment by kporras07** [comment]
This was fixed and 3.0.4 was just released. Thanks for reporting
